js/errors_test.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, assert } from "./test_util.ts";
+import * as deno from "deno";
+
+test(async function testErrorClasses() {
+  const num = Object.keys(deno.errorKinds).length / 2;
```

Comment:
Why `/ 2` ?

Reply:
An enum makes a bi-directional map between the key and values like this:
```
const enum = {};
enum[enum[0] = "Key"] = 0;
enum[enum[1] = "Foo"] = 1;
Object.keys(enum) // [0, "key", 1, "Foo"]
```